packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -670,6 +671,25 @@ const TRANSFORMS = {
     }
   },
 
+  sm(obj) {
+    return {
+      type: 'SM',
+      uuid: obj.uuid,
+      content_type: obj.content_type,
+      name_description: obj.name_description,
+      name_label: obj.name_label,
+
+      SM_type: obj.type,
+      tags: obj.tags,
+      configuration: obj.configuration,
+      vendor: obj.vendor,
+      features: obj?.features,
+      driver_filename: obj.driver_filename,
+      required_cluster_stack: obj.required_cluster_stack,
+      supported_image_formats: obj.supported_image_formats || [],
```

Comment:
Why testing all falsy value?
If you want to test `undefined/null` use `??` instead